### PR TITLE
Proposed Xcode string edits

### PR DESCRIPTION
**Pull request by DavidStaheli**
I'm proposing these string edits to bring the docs in-sync with the task.json.

### CODE REVIEW

File: Tasks/Xcode/task.json [hunk 1]
```diff
@@ -220,7 +221,7 @@
             "label": "Team ID",
             "defaultValue": "",
             "required": false,
-            "helpMarkDown": "Specify the 10 digit development Team ID. This is required if you are a member of multiple development teams.",
+            "helpMarkDown": "(Optional, unless you are a member of multiple development teams.) Specify the 10-digit development team ID.",
```
Comment (lkillgore) [inline Tasks/Xcode/task.json]: `10-digit` is not correct.  `10-character` instead.

File: Tasks/Xcode/task.json [hunk 2]
```diff
@@ -282,24 +283,24 @@
...
-        "FailedToGenerateExportOptionsPlist": "Failed to automatically generate an Export Options Plist to export an IPA. Please specify an Export Options Plist in the Package Options.",
-        "LocateXcodeBasedOnVersion": "Locate the developer path for Xcode %s",
-        "FailedToLocateSpecifiedXcode": "Unable to resolve the developer path for Xcode %s. Set the %s environment variable on the agent machine, or set `Xcode version` to `Default` or `Custom`.",
-        "CantDetermineProvisioningStyle": "Provisioning style not found in a pbxProject file. Defaulting to automatic signing. If the project uses manual signing, auto export may fail. `Export Options` can be set to `Specify` or `Plist` as a workaround."
+        "TempKeychainDeleteFailed": "Failed to delete the temporary keychain created during the build: `%s`",
+        "ProvProfileDeleteFailed": "Failed to delete the provisioning profile `%s`.",
+        "ExportMethodNotIdentified": "Failed to automatically identify the export method to use from the archive file. This might cause errors during the build or produce an invalid package. Set `Export Options` to `Plist` or `Specify`.",
```
Comment (lkillgore) [inline Tasks/Xcode/task.json]: I think `Export Options` is now `Export options`.

File: Tasks/Xcode/task.json [hunk 3]
```diff
@@ -282,24 +283,24 @@
...
+        "ExportOptionsPlistInvalidFilePath": "The `Export Options` plist file does not exist at `%s`. Provide the path to a valid plist file.",
+        "SchemeRequiredForArchive": "The scheme must be specified to generate the package with xcodebuild archive and export.",
+        "WorkspaceOrProjectRequiredForArchive": "The workspace or project path must be specified to generate the package with xcodebuild archive and export.",
+        "FailedToGenerateExportOptionsPlist": "Failed to automatically generate an `Export Options` plist file to export an IPA. Please specify an `Export Options` plist file in `Package Options`.",
```
Comment (lkillgore) [inline Tasks/Xcode/task.json]: `Package options`

File: Tasks/Xcode/task.json (hunk 2 above)
Comment (DavidStaheli) [reply Tasks/Xcode/task.json]: Done... thanks!

File: Tasks/Xcode/task.json (hunk 3 above)
Comment (DavidStaheli) [reply Tasks/Xcode/task.json]: Done... thanks!

File: Tasks/Xcode/Strings/resources.resjson/en-US/resources.resjson [hunk 4]
```diff
@@ -3,73 +3,74 @@
...
-  "loc.input.label.xcWorkspacePath": "Workspace/Project Path",
-  "loc.input.help.xcWorkspacePath": "Optional relative path from repo root to the Xcode workspace or project. For example: `MyApp/MyApp.xcworkspace` or `MyApp/MyApp.xcworkspace/MyApp.xcodeproj`. Leave blank if you intend to use the -target flag under Advanced Arguments.",
+  "loc.input.help.sdk": "Specify an SDK to use when building the Xcode project or workspace. From the macOS Terminal application, run `xcodebuild -showsdks` to display the valid list of SDKs. The default value of this field is the variable `$(SDK)`. When using a variable, make sure to specify a value (for example, `iphonesimulator`) on the **Variables** tab.",
+  "loc.input.label.xcWorkspacePath": "Workspace or project path",
+  "loc.input.help.xcWorkspacePath": "(Optional) Enter a relative path from the root of the repository to the Xcode workspace or project. For example: `MyApp/MyApp.xcworkspace` or `MyApp/MyApp.xcworkspace/MyApp.xcodeproj`. Do not specify a value if you use the `-target` argument under **Advanced** > **Arguments** (below).",
```
Comment (madhurig) [inline Tasks/Xcode/Strings/resources.resjson/en-US/resources.resjson]: We should remove this now since we support setting a project value here

**Pull request merged**